documentation/concepts/check.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: documentation
+menu: check
+subtitle: Check
+redirect_from: /documentation/check/
+---
+# Check
+{% include teams.html %}
+
+Check produces comparison reports to indicate differences between the current configuration of your database and what your migration scripts define. You might use this to get a summary of pending changes as part of an in-person review process or as a validation of the effect of all pending changes. 
```

Comment:
- Could this drop `comparison` and just be reports? People used to compare have found this confusing, and the fact that it's a comparison underneath is arguable an implementation detail that doesn't need to be exposed as far as the final report is concerned
- `current configuration` feels odd, since configuration typically refers to actual Flyway config or database config as opposed to the database schema. Could this be simplified to `your current database and what your migration scripts define` or something similar?

Reply:
Yup - agreed